src/jit/lsra.h

```diff
@@ -623,6 +676,11 @@ class LinearScan : public LinearScanInterface
     regMaskTP GetRegMaskForIndex(RegMaskIndex index);
     void RemoveRegisterFromMasks(regNumber reg);
 
+    static bool isSingleRegister(regMaskTP regMask)
```

Comment:
nit: seems like this should be a general helper located near `genMaxOneBit` called, e.g., `genExactlyOneBit`

Reply:
I'll add `genExactlyOneBit`, but I would like to keep this, as I think it is clearer. Note that this was pre-existing, but was defined in lsra.cpp instead of lsra.h.